core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel_Common.hpp

```diff
@@ -370,7 +371,7 @@ struct ParallelReduceSpecialize<FunctorType, TeamPolicyInternal<PolicyArgs...>,
     const size_t shmem_size_L0 = p.scratch_size(0, team_size);
     const size_t shmem_size_L1 = p.scratch_size(1, team_size);
     OpenMPTargetExec::resize_scratch(PolicyType::member_type::TEAM_REDUCE_SIZE,
-                                     shmem_size_L0, shmem_size_L1, league_size);
+                                     0, shmem_size_L1, league_size);
```

Comment:
Don't you still need the L0 allocation here (and below) if `KOKKOS_IMPL_OPENMPTARGET_LLVM_EXTENSIONS` isn't defined?

Reply:
Yes, thanks. Fixed it.